jenkins/model_output_processing_Jenkinsfile

```diff
@@ -28,18 +28,13 @@ pipeline {
         // May need to actually use yesterday's date depending on what time of day this runs
         //sh 'DATE=$(date -d "yesterday 13:00" "+%Y_%m_%d"); wget https://owi-common-resources.s3-us-west-2.amazonaws.com/resources/application/nhm/output/data/climate_${DATE}.nc'
         sh '''
-          PATTERN="*.nc"
+          sh 'aws s3 sync s3://owi-common-resources/resources/application/nhm/output/data/ . --exclude "*" --include "*.nc"'
```

Comment:
Is this sh within a sh?

Reply:
ah woops yes.

Reply:
Instead of using aws s3 sync and then wget, why not use the same command (aws s3 sync) for both (or aws s3 cp)

Reply:
good point, I can try both aws s3 sync or cp. i thought there was a reason why we were doing one versus the other but it eludes me now.